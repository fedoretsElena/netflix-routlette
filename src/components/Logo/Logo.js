import React from 'react';
import { Link } from "react-router-dom";

import './Logo.scss';

const Logo = () => {
  return <div className="logo f-size-18"> <Link to="/"><b>netflix</b>roulette</Link></div>
}

export default Logo;