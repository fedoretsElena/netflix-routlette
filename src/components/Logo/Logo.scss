@import './../../styles/variables';

.logo {
  letter-spacing: .7px;

  a {
    color: $primary;
    text-decoration: none;
  }

  a:hover {
    text-decoration: none;
  }
}