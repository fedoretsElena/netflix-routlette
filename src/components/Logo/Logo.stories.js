import React from 'react';
import { BrowserRouter } from 'react-router-dom';

import Logo from './Logo';

export default {
  component: Logo,
  title: 'Logo',
};

const Template = args => <BrowserRouter>
  <Logo {...args} />
</BrowserRouter>;

export const Default = Template.bind({});