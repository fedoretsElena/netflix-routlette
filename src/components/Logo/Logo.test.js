import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router';

import Logo from './Logo';

describe('Logo component', () => {
  it('should render Logo snapshot', () => {
    const { asFragment } = render(<MemoryRouter><Logo/></MemoryRouter>);
    expect(asFragment(<MemoryRouter><Logo/></MemoryRouter>)).toMatchSnapshot();
  })
})