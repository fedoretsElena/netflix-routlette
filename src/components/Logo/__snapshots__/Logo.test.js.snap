// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component should render Logo snapshot 1`] = `
<DocumentFragment>
  <div
    class="logo f-size-18"
  >
     
    <a
      href="/"
    >
      <b>
        netflix
      </b>
      roulette
    </a>
  </div>
</DocumentFragment>
`;
